### PR TITLE
Shelve Pixel Demo

### LINKED ISSUE #506: [pixel-demo, cosmic-swingset, agoric-cli, documentation] Shelve the Pixel Demo For Now
Our current priority is getting an MVP of ERTP and Zoe on the testnet with developer tools like the agoric-cli to create contracts and apps and deploy them. 

The pixel-demo doesn't help us do that, and it needs a significant amount of attention to make it a good experience. We would need to rewrite it to use Zoe rather than the contractHost (#495) for example, in addition to UI work. Thus, we decided to temporarily shelve it until it can get the attention that it needs. In its place, we will promote the wallet frontend and autoswap frontend and encourage people to interact with contracts running on Zoe

We recently moved the pixel-demo into its own package in the mono-repo, taking it out of ERTP. Shelving would mean leaving the package in the mono-repo, but snipping off the imports in cosmic-swingset. We can also move any pixel-demo configs to the pixel-demo package, since we aren't trying to support it in ERTP any longer.

### PR DESCRIPTION
**Pull request by katelynsills**
Closes #506 

This PR removes the ERTP and cosmic-swingset dependencies on `@agoric/pixel-demo`. The pixel-demo code still remains as is to be picked up later, except for skipping the pixel-demo tests. 

In ERTP, this PR removes the pixel-demo extentOps from the approved collection of extentOps. This eliminates the circular dependency of ERTP depending on pixel-demo, which depends on ERTP.

In cosmic-swingset, this PR removes vat-pixel.js, removes the bootstrap steps to create the user bundle for the pixel demo, and removes the front-end display of the pixel demo. I think we should be able to look back at this commit in the future to regain the front-end code if we need it. 

Future steps:
* We will need to connect the autoswap and wallet frontend to non-agoric-cli uses of cosmic-swingset to replace the pixel demo, but I'll make a separate issue for that. 
* Also, the README in cosmic-swingset is pretty old and should probably be rewritten and moved to agoric.com/documentation.

### CODE REVIEW

**Comment by katelynsills** [comment]
Also incidentally closes #252

**Review by michaelfig — COMMENTED** [review]
I don't want to land this PR until we have a candidate for the Wallet/Autoswap demo.

If you'd still like me to review in advance, I can, but would prefer to wait until then.

**Comment by katelynsills** [comment]
> I don't want to land this PR until we have a candidate for the Wallet/Autoswap demo.

Ok, thanks. What does having a candidate for the Wallet/Autoswap demo mean? You would like for #509 to be solved in this PR? #509 is outside my expertise since I haven't done any of the wiring so I would need help from someone else.

**Comment by michaelfig** [comment]
> Ok, thanks. What does having a candidate for the Wallet/Autoswap demo mean? You would like for #509 to be solved in this PR?

Doesn't need to be solved here, but I'd like to at least know we're close to landing a PR for it.  I want to lay the groundwork (such as #501) before we whip the carpet out from under the testnet's demo. 😉

**Comment by michaelfig** [comment]
Okay, I agree with where you're coming from.

Would it be okay to ~~agree on~~ and implement a solution for #501 (~~which could use your feedback~~ just saw your feedback), and then I'll review and approve this PR?  Being able to have steps for publishing a contract to Zoe on the Testnet is what I'd consider the only really essential thing that would make me comfortable ditching the pixel demo.  That would be my new smoke test.

**Comment by katelynsills** [comment]
> Would it be okay to agree on and implement a solution for #501 (which could use your feedback), and then I'll review and approve this PR? Being able to have steps for publishing a contract to Zoe on the Testnet is what I'd consider the only really essential thing that would make me comfortable ditching the pixel demo. That would be my new smoke test.

Sure, that sounds great to me. Thank you for doing that!

**Review by michaelfig — APPROVED** [review]
LGTM!

I'm still nervous about landing this before #501, but I'll leave it to your discretion.  I can always branch from before the landing if we really need a testnet with the pixel demo.

File: packages/cosmic-swingset/README.md
Comment (Chris-Hibbert) [inline packages/cosmic-swingset/README.md:283]: We will still make a moola[h] purse, I think. Does it not end up in `home`?

` assetNames = ['moola', 'simolean'];`

Comment (katelynsills) [reply packages/cosmic-swingset/README.md:283]: The wallet has all the purses now!

**Review by Chris-Hibbert — APPROVED** [review]

**Comment by katelynsills** [comment]
Ok, thanks! I'll leave this up until tomorrow to try to land #501 first, but if there's a delay let's still try to get this in tomorrow sometime.

File: packages/pixel-demo/test/swingsetTests/gallery/test-gallery.js
Comment (warner) [inline packages/pixel-demo/test/swingsetTests/gallery/test-gallery.js:27]: Why keep these around? I guess I think of `test.skip` as a temporary bandage while something is known to fail, that will eventually graduate into a normal `test` (so we can write the tests first, then implement the feature). Is this `test.skip` because we think we might bring back this functionality?

**Review by warner — COMMENTED** [review]
Am I correct in understanding that the new `cosmic-swingset` experience (when you run scenario 2 or 3 and then visit `http://localhost:8080/` will have a REPL box that you can interact with, just without the pixel display? That's not as interesting to look at, but would still provide enough functionality for the sort of manual testing that I care about.

**Comment by warner** [comment]
+1 overall

**Comment by katelynsills** [comment]
> Am I correct in understanding that the new `cosmic-swingset` experience (when you run scenario 2 or 3 and then visit `http://localhost:8080/` will have a REPL box that you can interact with, just without the pixel display? That's not as interesting to look at, but would still provide enough functionality for the sort of manual testing that I care about.

Yup, that's exactly right. The gallery box is removed and the cli is moved over so that it takes up the whole width for now.

File: packages/pixel-demo/test/swingsetTests/gallery/test-gallery.js
Comment (katelynsills) [reply packages/pixel-demo/test/swingsetTests/gallery/test-gallery.js:27]: Yeah, I'm pretty sure we want to bring this example back, just after Zoe Alpha. So I think it fits the test.skip temporary bandage concept - it fails now, we know why, we know we need to redesign the feature, but we can't do it right now.

**Pull request merged**
